tools/check_constitution.py

```
@@ -0,0 +1,450 @@
+            continue  # accepted at the merge base: immutable, and past repair
+        text = f.read_text(encoding="utf-8")
+        # Three sites name one entry. Left free to disagree, [D-N] resolves to
+        # whichever the reader happened to consult.
+        extra = [x for x in text.split(chr(10)) if STATUS_LINE.match(x)]
+        if len(extra) > 1:
+            findings.append(
+                f"entry-shape (§12): {rel} has {len(extra)} `**Status:**` lines — repair it "
+                f"before it lands; afterwards the record can be neither superseded nor fixed"
+            )
+        own = ENTRY_NAME.match(f.name).group(1)
+        for pat, site in ((rf"^# D-(\d+):", "heading"),
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)